components/forms/Donate/Donateguild.tsx

```diff
@@ -9,15 +9,15 @@ const stepThree = ' covers an entire month of therapy sessions.'
 const stepFour = ' covers an entire month of therapy sessions for two people.'
 
 export default function DonateGuild() {
-  const [loading, setLoading] = useState(false)
-  const [input, setInput] = useState()
-  const [impact, setImpact] = useState('$30')
-  const [message, setMessage] = useState(stepTwo)
+  const [loading, setLoading] = useState<boolean>(false)
+  const [input, setInput] = useState({ value: 0 })
+  const [impact, setImpact] = useState<string>('$30')
+  const [message, setMessage] = useState<string>(stepTwo)
```

Comment:
Bit pf a nitpick, but I think all the generics here can be inferred

```suggestion
  const [loading, setLoading] = useState(false)
  const [input, setInput] = useState({ value: 0 })
  const [impact, setImpact] = useState('$30')
  const [message, setMessage] = useState(stepTwo)
```

Reply:
Implemented